.github/workflows/pull_request_merger.yml

```diff
@@ -0,0 +1,18 @@
+name: pull-request-merger
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+          fetch-depth: '50'
+
+      - name: Skywater PDK Libraries Pull Request Merger on Release Branches
+        uses: agorararmard/skywater-pdk-modules-pull-request-backporter-action@v2
```

Comment:
```suggestion
        uses: google/skywater-pdk-modules-actions/modules-pr-backporter@main
```

Reply:
as per our discussion two weeks ago, this PR is obsolete. We will be getting the history for this repo from: https://github.com/efabless/skywater-pdk-libs-sky130_fd_sc_hd